apps/backend/src/middleware.tsx

```diff
@@ -54,8 +54,7 @@ export async function middleware(request: NextRequest) {
   const isApiRequest = url.pathname.startsWith('/api/');
 
   const newRequestHeaders = new Headers(request.headers);
-  // store the direct IP address of the requester or proxy so we can read it with `headers()` later
-  newRequestHeaders.set("x-stack-direct-requester-or-proxy-ip", request.ip ?? '');
```

Comment:
What was this for and why don't we need it anymore?

Reply:
It was for the geolocation stuff, Next 15 removed `request.ip` so now we just don't check it anymore (we still check various header as a fallback)